src/lib/rust_px4_nuttx/src/nuttx/time.rs

```diff
@@ -0,0 +1,57 @@
+use core::ops::Add;
+use core::ops::AddAssign;
+
+extern "C" {
+    fn hrt_absolute_time() -> u64;
+}
+
+pub fn hrt_time() -> u64 {
+    unsafe {
+        hrt_absolute_time()
+    }
+}
+
+pub struct Duration(u64);
```

Comment:
Is this strictly needed as there is also [`core::time::Duration`](https://doc.rust-lang.org/stable/core/time/struct.Duration.html) available?

Reply:
No real reason to use own struct. Better to use one from corelib

Reply:
Implementation of rust_px4_nuttx lib is moved into it's own repo